web/src/enterprise/campaigns/detail/CampaignDetails.tsx

```diff
@@ -87,7 +93,7 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
     const [name, setName] = useState<string>('')
     const [description, setDescription] = useState<string>('')
     const [type, setType] = useState<CampaignType>()
-    const [campaignPlanArguments, setCampaignPlanArguments] = useState<string>('')
+    const [campaignPlanArguments, setCampaignPlanArguments] = useState<string | undefined>()
```

Comment:
The `| undefined` is implicit and correctly inferred when an argument for the initial value is not passed.

```suggestion
    const [campaignPlanArguments, setCampaignPlanArguments] = useState<string>()
```